components/global/Header/Navigation.js

```diff
@@ -165,7 +163,7 @@ Navigation.propTypes = {
   items: PropTypes.arrayOf(internalLinkWithChildrenShape),
   userProfilePage: PropTypes.object,
   theme: PropTypes.oneOf(["desktop", "mobile"]),
-  isOpenSetter: PropTypes.func,
+  desktopSetter: PropTypes.func,
```

Comment:
renamed to `desktopSetter` to clarify that it is only impactful to the desktop nav.